src/order/complete_lattice.lean

```diff
@@ -322,6 +331,15 @@ lemma monotone.map_infi2_le [complete_lattice β] {f : α → β} (hf : monotone
 calc f (⨅ i (h : ι' i), s i h) ≤ (⨅ i, f (⨅ h, s i h)) : hf.map_infi_le
 ... ≤ (⨅ i h, f (s i h)) : infi_le_infi $ λ i, hf.map_infi_le
 
+lemma infi_comp_ge {ι' : Sort*} (f : ι' → α) (g : ι → ι') :
```

Comment:
Why not `le_infi_comp`, following the order of symbols in the statement?

Reply:
No reason, I'll rename.

Reply:
Renamed this and `monotone.map_supr(2)_ge`, waiting for CI.